soccer/gameplay/plays/testing/test_capture.py

```diff
@@ -33,8 +34,13 @@ def __init__(self):
             'robot away from ball')
 
         self.add_transition(
-            TestCapture.State.capturing, TestCapture.State.setup,
+            TestCapture.State.capturing, TestCapture.State.rotating,
             lambda: self.subbehavior_with_name('capture').state == behavior.Behavior.State.completed,
+            'rotating to check capture')
+
+        self.add_transition(
+            TestCapture.State.rotating, TestCapture.State.setup,
+            lambda: self.subbehavior_with_name('aim').state == skills.aim.Aim.State.aimed,
```

Comment:
If aim goes into the failed state, this play will be stuck in this state and won't do anything.